minecraft_servers/pvparmy/manifest.json

```diff
@@ -1,13 +1,15 @@
 {
   "server_name": "pvparmy",
   "nice_name": "PvPArmy",
-  "direct_ip": "pvparmy.de",
+  "direct_ip": "pvparmy.net",
   "server_wildcards": [
+    "%.pvparmy.net",
     "%.pvparmy.de"
   ],
   "social": {
-    "web": "https://pvparmy.de",
+    "web": "https://pvparmy.net",
```

Comment:
New website is also in maintenance. Please remove the line to prevent a redirect to a maintenance-website.